yb-voyager/cmd/exportData.go

```diff
@@ -176,6 +176,30 @@ func exportData() bool {
 		utils.ErrExit("failed to add the leaf partitions in table list: %w", err)
 	}
 
+	leafPartitions := make(map[string][]string)
```

Comment:
What display problem are we solving here exactly?

Reply:
The same with the table list we display on UI for users to see what is being exported. As the actual table list is with leaf partitions but to display we should show the root as per what we had decided.